x/ecocredit/basket/codec.go

```diff
@@ -0,0 +1,17 @@
+package basket
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/types/msgservice"
+)
+
+func RegisterTypes(registry codectypes.InterfaceRegistry) {
+	msgservice.RegisterMsgServiceDesc(registry, &_Msg_serviceDesc)
+}
```

Comment:
we should add here the message interfaces as well.

Reply:
Pretty sure that's taken care of automatically by this function